src/common/search/FilterPlays.jsx

```diff
@@ -201,11 +206,13 @@ const filterObject = {
 const FilterPlays = ({ reset }) => {
   const location = useLocation();
   const navigate = useNavigate();
-  const { setFilterQuery } = useContext(SearchContext);
+  const { filterQuery, setFilterQuery } = useContext(SearchContext);
   const [showModal, setShowModal] = useState(false);
   const [modifiedFilterQuery, setModifiedFilterQuery] = useState({ ...filterObject });
   const [noOfAppliedFilter, setnoOfAppliedFilter] = useState(0);
 
+  const [ifFilterApplied, setIfFilterApplied] = useState(false);
```

Comment:
for boolean states, starting the state name with verbs like `is` or `has` is more convient since it conveys the meaning clearly.

something like:
- `isFilterApplied`

Reply:
Got it.
I will rename the state variable